src/libraries/Microsoft.Extensions.Logging.Abstractions/src/LogEntry.cs

```diff
@@ -19,7 +19,7 @@ namespace Microsoft.Extensions.Logging.Abstractions
         /// <param name="state">The state for which log is being written.</param>
         /// <param name="exception">The log exception.</param>
         /// <param name="formatter">The formatter.</param>
-        public LogEntry(LogLevel logLevel, string category, EventId eventId, TState state, Exception exception, Func<TState, Exception, string> formatter)
+        public LogEntry(LogLevel logLevel, string category, EventId eventId, TState state, Exception? exception, Func<TState, Exception?, string>? formatter)
```

Comment:
TState seems it may be null. Double check with the documentation

Reply:
It's up to the caller whether they want to specify a nullable or non-nullable TState.